tf-psa-crypto/core/common.h

```diff
@@ -0,0 +1,435 @@
+/**
```

Comment:
> The common.h file is copied instead of
being just moved as eventually they
will be different in mbedtls and
TF-PSA-Crypto. Some parts of it can be
shared though but this will be done later,
probably when adding the CMake build
system in tf-psa-crypto.

_sigh_ Why? Why isn't `library/common.h` just `#include "core/common.h"`? Or just omit `library/common.h` since we don't actually have anything to put there? Why are we spending effort on keeping two files identical? And how will that work when they're no longer in the same repository?